### PR TITLE
add crane to wolfi os

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>
Co-authored-by: Furkan Türkal <furkan.turkal@trendyol.com>

> https://github.com/google/go-containerregistry/tree/main/cmd/crane

/cc @imjasonh @kaniini @dlorenc

### CODE REVIEW

File: crane.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    repositories:
+      - https://packages.wolfi.dev/os
```
Comment (tuananh) [inline crane.yaml:21]: i think you need to remove repo & keyring. they are appended in ci script

Comment (developer-guy) [reply crane.yaml:21]: So many packages are already using the keyring; this is why I added it too 👇

> https://github.com/search?q=repo%3Awolfi-dev%2Fos+wolfi-signing.rsa.pub&type=code

Comment (kaniini) [reply crane.yaml:21]: Yeah it was a mistake that those entires were added.

Comment (developer-guy) [reply crane.yaml:21]: do you want me to remove the keyring ?

**Comment by developer-guy** [comment]
I pulled the archive to my local workstation then I ran `sha256sum` on it, I gave the result into the checksum field of the fetch pipeline, but still, it's not working :/

```shell
httpie --download https://github.com/google/go-containerregistry/archive/refs/tags/v0.12.1.tar.gz
$ sha256sum v0.12.1.tar.gz
6f8060933ace2acff468ce17359aa858b7ca3db049ed8d0ac5d4ae62359573c3  go-containerregistry-0.12.1.tar.gz
```

**Review by kaniini — APPROVED** [review]

**Pull request merged**